requirements.txt

```diff
@@ -1,16 +1,17 @@
-blinker==1.4
-dominate==2.1.17
-Flask==0.10.1
 Flask-Bootstrap==3.3.5.7
```

Comment:
Sorted this file before I realized it would come into the repo as a bunch of changes.